### ISSUE TITLE
Localize connectors

### ISSUE THREAD

**Issue by adomasven** [zotero/zotero-connectors#58]

**Comment by dstillman** [comment]
See note at https://github.com/zotero/zotero/issues/798

**Comment by dstillman** [comment]
This is a bit more important with the target selector, since it's jarring to see "My Library" if you expect to see "Ma bibliothèque". We can move the actual strings gradually, but we should get a `Zotero.getString()` in place that we can use for selected things.

**Comment by dstillman** [comment]
Ugh. Much easier in terms of Transifex if the files are in the same place, but doing cross-repo string changes would be even more annoying, so I guess `src/locales`, and I'll deal with it somehow.

**Comment by dstillman** [comment]
(Actually, I'm pretty sure it's showing "Ma bibliothèque" now, since it gets that from the client, but there's still more UI to interact with now, so we should do this.)

**Comment by adomasven** [comment]
Actually, let's keep this open until we have tests (currently waiting on @dstillman breakage from target-selector to be fixed).

**Comment by gracile-fr** [comment]
> No localisation support in HTML. BrowserExt has native support and we
>   could add safari support with some gulp processing of html files if
>   need be

Do you mean that it's not possible at the moment to localize <a href="https://github.com/zotero/zotero-connectors/blob/master/src/common/preferences/preferences.html">preferences.html</a>, <a href="https://github.com/zotero/zotero-connectors/blob/master/src/common/itemSelector/itemSelector.html">itemSelector.html</a>, etc. ?

**Comment by adomasven** [comment]
> Do you mean that it's not possible at the moment to localize preferences.html, itemSelector.html, etc.?

It is possible, but you cannot insert [placeholders](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Internationalization) like `__MSG_general_savingTo__` in HTML files and strings instead have to be localised using JS and via `Zotero.getMessage()`. A lot of the preferences strings are already only defined in preferences.js, so it's not a big deal.

**Comment by Elaborendum** [comment]
Are there any updates on this? It's been quite a while.

**Comment by dstillman** [comment]
@Elaborendum: This is already done, though not everything is localized currently. If there's something specific you're referring to, best to post to the forums with your question.

**Comment by dstillman** [comment]
We can leave this open until more of the UI has been transferred to localized strings.
